Source/BugsnagSessionTrackingPayload.h

```diff
@@ -10,9 +10,11 @@
 
 #import "BugsnagSession.h"
 
+@class BugsnagConfiguration;
+
 @interface BugsnagSessionTrackingPayload : NSObject
 
-- (instancetype)initWithSessions:(NSArray<BugsnagSession *> *)sessions;
+- (instancetype)initWithSessions:(NSArray<BugsnagSession *> *)sessions config:(BugsnagConfiguration *)config;
```

Comment:
nit: looks better formatted over two lines

Reply:
Formatted all instances in this file like this, marked the rest as resolved.